netctl/netctl.go

```diff
@@ -401,3 +402,14 @@ func dumpList(ctx *cli.Context, list []map[string]interface{}) {
 	os.Stdout.Write(content)
 	os.Stdout.WriteString("\n")
 }
+
+func showVersion(ctx *cli.Context) {
```

Comment:
agree; we could add the `k8contiv -version` command separately, and same for `netctl version` however the main reason for someone to issue `netctl version` was to show the netmaster/netplugin version, so perhaps keep it under the heading `netmaster's verison` and then show netmaster version